src/main/java/org/folio/sidecar/service/TenantService.java

```diff
@@ -113,11 +114,11 @@ private static List<String> getTenantIds(ResultList<Entitlement> resultList) {
   private void addEnabledTenants(List<Tenant> tenants) {
     enabledTenants.clear();
     tenants.forEach(tenant -> enabledTenants.add(tenant.getName()));
-    log.info("Module is enabled for tenants: {}", tenants);
+    log.info("Module is enabled for tenants: {}", tenants.stream().map(Tenant::getName).toList());
```

Comment:
when something should be computed it's better to wrap the computation in a lambda to avoid unnecessary evaluation when the debug level is higher than `info.`